api/v3/CustomValue/Getallgroupstreevalues.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * @file
+ * API file.
+ */
+
+/**
+ * CustomValue.getallgroupstreevalues API specifications.
+ *
+ * @param array $spec
+ *   Description of fields supported by this API call.
+ */
+function _civicrm_api3_custom_value_getallgroupstreevalues_spec(array &$spec) {
```

Comment:
Not too sure about the name for this new API. Can we try `getalltreevalues` or something better since it returns for both inline and tabs custom groups.